### ISSUE TITLE
GeoDCAT-AP

### ISSUE THREAD

**Issue by bertvannuffelen** [opengeospatial/geosemantics-dwg#105]
Dear WG,

I am supporting SEMIC as editor for the DCAT profiles. I heard that OGC is willing to invest in an update on GeoDCAT-AP. 
I reach out to streamline this activity.

kr,

Bert

**Comment by lvdbrink** [comment]
Sorry to wait this long to reply. We are working on a working group charter as you can see in [PR 106](https://github.com/opengeospatial/geosemantics-dwg/pull/106). We will be glad to have your input on this draft charter. 

Is there a SEMIC DCAT profile in the makings for geospatial data? Perhaps we should have an adhoc meeting to coordinate. 

Else you can join the GeoDCAT working group once it's formed or if you're not an OGC member we'll see how you can contribute. 

@rob-metalinkage what do you think?

**Comment by rob-metalinkage** [comment]
Perfect.  Note proposal to separate -AP specific concerns from general DCAT as required.

**Comment by dr-shorthair** [comment]
The most recent SEMIC GeoDCAT profile appears to be here https://semiceu.github.io/GeoDCAT-AP/releases/

**Comment by bertvannuffelen** [comment]
@lvdbrink my apologizes for a later reply.

The reference that @dr-shorthair  has provided is indeed correct. 

 I believe SEMIC is not an official member of OGC in some way. So lets have an adhoc meeting.  What is the best way to find a slot?
 Do I post a doodle link here?

kr,

Bert

**Comment by bertvannuffelen** [comment]
@lvdbrink and @dr-shorthair sorry to bother you via this way, just a quick reminder to reach out for an adhoc meeting.

**Comment by rob-metalinkage** [comment]
would be good to discuss the SEMIC approach here:

"The namespace for GeoDCAT-AP is: http://data.europa.eu/930/

The suggested namespace prefix is: geodcat"

OGC/W3C will want to publish a GeoDCAT unencumbered by AP (european) considerations, that can be specialised into GeoDCAT-AP - i guess we can opt for gdcat as a prefix to avoid clashes.

we will aim for machine-readability however..

note 
 
curl -LH  "Accept: text/turtle" https://data.europa.eu/930/  

just gets the HTML again.

and accessing this has quite an exciting list of redirects:

```
Status Code	URL	IP	Page Type	Redirect Type	Redirect URL	
301	http://data.europa.eu/930/	13.35.138.43	server_redirect	permanent	https://data.europa.eu/930/
307	https://data.europa.eu/930/	13.35.138.43	server_redirect	temporary	http://uri.semic.eu/geodcatap/
308	http://uri.semic.eu/geodcatap/	63.32.50.253	server_redirect	permanent	https://uri.semic.eu/geodcatap/
302	https://uri.semic.eu/geodcatap/	63.32.50.253	server_redirect	temporary	https://semiceu.github.io/GeoDCAT-AP/releases/
200	https://semiceu.github.io/GeoDCAT-AP/releases/	185.199.110.153	normal	none	none
```
it may not be feasible to mix http and https for a client attempting to access machine readable content - need to check on this.

**Comment by bertvannuffelen** [comment]
@rob-metalinkage no problem to discuss also the publication expectations and to work on that. 
My first objective is to clarify the governance. So that we are reinforcing each other.

**Comment by rob-metalinkage** [comment]
@bertvannuffelen agree governance is critical.  It makes sense for OGC to deal with spatio-temporal descriptions,  (OWL-time is a joint OGC/W3C standard) - and SEMIC to deal with GeoDCAT-AP as a profile of both GeoDCAT and DCAT-AP - this is clear governance to support reusability in the wider context and a formal process that the wider community can engage with. SEMIC of course is invited and encouraged to directly engage in the governance of GeoDCAT via the proposed OGC working group to ensure its backwards-compatibility concerns are adequately captured and tested.

**Comment by lvdbrink** [comment]
> @lvdbrink and @dr-shorthair sorry to bother you via this way, just a quick reminder to reach out for an adhoc meeting.

Yes, I'm happy to create a doodle. Just to get a feeling - is it feasible to start looking for a date in the week of Nov 14?

**Comment by TallTed** [comment]
@rob-metalinkage — It would be helpful if you could edit your https://github.com/opengeospatial/geosemantics-dwg/issues/105#issuecomment-1288622437, and put code fences (lines of just three backticks, in this case, above and below) around the terminal interactions.  This will especially make the table of redirects easier to digest!

**Comment by bertvannuffelen** [comment]
@all, 

here is the doodle poll for the week of the 14th and the 22th of November. I took blocks of 2 hours to make the doodling shorter. I expect a 1 hour meeting will be sufficient.
https://doodle.com/meeting/participate/id/dyoP6m7a

**Comment by bertvannuffelen** [comment]
@all,

we have a shared moment the 22th Nov 2022 9-11 Brussels time. I will post a link to MS teams before the meeting here.

**Comment by lvdbrink** [comment]
Actually, I think it is best to use OGC teleconferencing faciliities for this. I created a GoToMeeting session for 9AM-10Am CET using the OGC facilities. Anyone with OGC login can find it on the portal in the calendar. I will also announce it on the GeoSemantics mailing list and forward it to MetaCat DWG. I'm a bit hesitant to share the link here, so please send me an email if you can't accesss the OGC portal and are not a member of the GeoSemantics /MetaCAT DWG mailing list.

**Comment by lvdbrink** [comment]
@ALL

I published the meeting notes in this github repository: https://github.com/opengeospatial/geosemantics-dwg/blob/master/geodcat-adhoc/20221122-adhoc-notes.md

Corrections and additions are welcome! Preferably via github.

**Comment by lvdbrink** [comment]
@ALL 

I created a Doodle to pick the meeting date & time for the second ad hoc session. https://doodle.com/meeting/participate/id/e3QPoj4b

I took 2 hour blocks to make it easier to create the doodle, but 1 hour should be enough, again. Please, anyone who wants to join, could you indicate your availability?

**Comment by lvdbrink** [comment]
Based on the Doodle results the next meeting is: 
Dec 15, 2022, 2:00 PM - 3:00 PM Europe/Zurich (GMT+01:00)

Like the last time I will create an OGC calendar event including GoToMeeting facilicties will send it to anyone that can't access the OGC portal.
